desk/app/groups.hoon

```diff
@@ -1314,14 +1312,18 @@
   =^  caz=(list card)  subs
     (~(subscribe s [subs bowl]) wire dock path delay)
   (emil caz)
-::  +safe-force-watch: safely force watch a subscription path
+::  +safe-unwatch: safely unsubscribe from a path
```

Comment:
It's not really "safe" in the way that "safe-watch" is "safe", so the naming here is a little confusing. Maybe better to do "eager-leave" or "proactive-leave" or something in that direction.